### PR TITLE
Feature/form fields disabled state

### PR DESCRIPTION
**Pull request by tom-odb**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Custom form controls have no disabled state.

Issue Number: 132

## What is the new behavior?
Added disabled state for all form controls (except upload, the input component needs to be revised).

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Resolved issues
Resolves: #132

### CODE REVIEW

File: packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 		(keyEscape)="onKeyEscape()"
 		(focus)="onFocus()"
 		autocomplete="off"
+		[attr.disabled]="isDisabled ? true : null"
```
Comment (TriangleJuice) [inline packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html]: Mag `attr` hier niet gewoon weg?

File: packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html [hunk 2]
```diff
@@ -29,9 +30,10 @@
 		(focus)="onFocus()"
 		autocomplete="off"
 		[auiMask]="mask"
+		[attr.disabled]="isDisabled ? true : null"
 	/>
 
-	<div auiFlyoutZone>
+	<div auiFlyoutZone *ngIf="!isDisabled">
```
Comment (TriangleJuice) [inline packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html]: Idem.

File: packages/forms/lib/src/range-slider/components/range-slider/range-slider.component.scss [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
-    position: relative;
+	position: relative;
+
+	&.is-disabled {
+		cursor: not-allowed;
+
+		.m-range-slider__handle:hover {
+			cursor: not-allowed;
+		}
+	}
```
Comment (TriangleJuice) [inline packages/forms/lib/src/range-slider/components/range-slider/range-slider.component.scss]: Volgens mij zit alle indenting hier niet helemaal ok.

File: packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html (hunk 1 above)
Comment (tom-odb) [reply packages/forms/lib/src/auto-complete/components/auto-complete/auto-complete.component.html]: klopt, met al dat reactive/template gedoe door elkaar begon het verwarrend te worden 😛

**Pull request merged**